README.md

```diff
@@ -1,12 +1,61 @@
-# Shopping List
+![Bluemix Deployments](https://metrics-tracker.mybluemix.net/stats/5c5df69e10058d49cdc1f4d2fc63ce31/badge.svg)
 
-**Note:** This is a work in progress.
+# Create an Offline First Shopping List with Polymer and PouchDB
 
-Shopping List is an Offline First demo Progressive Web App built using Polymer and PouchDB. Mult-user / multi-device capabilities are enabled by Hoodie. [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list)
+In this developer journey, we will create an Offline First shopping list. Shopping List is an Offline First demo Progressive Web App built using Polymer and PouchDB. [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list) Credit goes to [Bradley Holt](https://github.com/bradley-holt) for this Polymer/PouchDB implementation.
 
-## Quick Start
+When the reader has completed this journey, they will understand how to:
 
-### Prerequisites
+* Implement an app database that runs in the browser with local data when offline, and syncs with a remote database when online.
+* Create an Offline First Progressive Web App.
+
+![](doc/source/images/architecture.png)
+
+## Flow
+
+1. The user manages shopping lists using the Offline First web app.
+2. Polymer is used to make it a Progressive Web App with Web Components and Service Workers.
+3. The shopping lists are stored locally in PouchDB while offline.
+4. When online, PouchDB syncs with Cloudant.
+
+## With Watson
+Want to take your Watson app to the next level? Looking to leverage Watson Brand assets? Join the [With Watson](https://www.ibm.com/watson/with-watson) program which provides exclusive brand, marketing, and tech resources to amplify and accelerate your Watson embedded commercial solution.
+
+## Included components
+* [Cloudant NoSQL DB](https://console.ng.bluemix.net/catalog/services/cloudant-nosql-db): A fully managed data layer designed for modern web and mobile applications that leverages a flexible JSON schema.
+
+## Featured technologies
+* [Polymer](https://www.polymer-project.org/): Libraries, tools, and patterns for building Progressive Web Apps using web platform features such as Web Components, Service Workers, and HTTP/2.
+* [PouchDB](https://pouchdb.com/): An open source JavaScript database that syncs with anything that implements the CouchDB Replication Protocol.
+* [Databases](https://en.wikipedia.org/wiki/IBM_Information_Management_System#.22Full_Function.22_databases): Repository for storing and managing collections of data.
```

Comment:
I'm not sure what this part means, exactly.

Reply:
If you mean specifically the desc of "Databases". I agree. That's our standard blurb from. Otherwise we use the components list for IBM stuff and featured tech for other stuff. So I like to use this to at least call out Polymer and PouchDB. I ended up moving other stuff to "Links" at the bottom.